src/pyrokinetics/pyro.py

```diff
@@ -1695,7 +1723,10 @@ def load_local_species(self, psi_n: float, a_minor: Optional[float] = None) -> N
             )
 
         if a_minor is not None:
-            self.norms.set_lref(minor_radius=a_minor)
+            if isinstance(a_minor, PyroQuantity):
+                self.norms.set_lref(minor_radius=a_minor)
+            else:
+                raise ValueError("a_minor must be specified with units")
```

Comment:
```suggestion
            if not isinstance(a_minor, PyroQuantity):
                raise ValueError("a_minor must be specified with units")
            self.norms.set_lref(minor_radius=a_minor)          
```

Ok, I get this is super minor, but I think it's a good habit to write checks this way round!